### ISSUE TITLE
Add IDs to navigation sidebar

### ISSUE THREAD

**Issue by msavy** [apiman/apiman#1559]
The sidebar in the main UI does not have IDs on link elements, we should add those to ensure they are more easily resolvable by our tooling (e.g. Cypress, screen readers, etc).

Acceptance criteria:
- All sidebar links should have a sensible ID (ensure this does not clash with the main ID). For example, `#sidebar-(same name as main ID)` could make sense.
